packages/airnode-node/src/evm/fulfillments/withdrawals.ts

```diff
@@ -78,7 +78,9 @@ export async function submitWithdrawal(
   // We set aside some ETH to pay for the gas of the following transaction,
   // send all the rest along with the transaction. The contract will direct
   // these funds back to the sponsor wallet.
-  const txCost = paddedGasLimit.mul(options.gasPrice);
+  const txCost = paddedGasLimit.mul(
+    options.gasPrice.gasPrice ? options.gasPrice.gasPrice : options.gasPrice.maxFeePerGas!
```

Comment:
Nit: The nested "gasPrice.gasPrice" isn't great 😅 Maybe try to rename the object?

Reply:
I appreciate the nits, how else will I learn :)
I'll rename the base gasPrice variable to gasTarget perhaps? `gasPrice` is a field in a type specified by `ethers`.